### ISSUE TITLE
Github Action failing to create JOSE draft

### ISSUE THREAD

**Issue by AlexanderJuestel** [openjournals/openjournals-draft-action#11]
Hello, 

I am trying to create a draft for the JOSE Journal using the following workflow adapted from https://github.com/marketplace/actions/open-journals-pdf-generator: 

https://github.com/cgre-aachen/gemgis_data/blob/main/.github/workflows/draft-pdf.yml

I copied the workflow and adapted it from a JOSS paper where the workflow worked as planned. However, with JOSE it is failing, see for example https://github.com/cgre-aachen/gemgis_data/actions/runs/2084732180

Some advice would be highly appreciated on what is wrong.
Have a good weekend and thanks in advance.

Cheers Alex

**Comment by tarleb** [comment]
Thanks for the report, I'll take a look.

**Comment by tarleb** [comment]
I identified two issue:

- There are some spurious dashes and extra-spaces in the metadata, which confuse the YAML parser.
- The use of BibTeX commands like `\citep` is not supported; replacing these with Markdown markup allows the paper to compile.

I'll open a PR to your repo with the necessary fixes.

**Comment by AlexanderJuestel** [comment]
@tarleb I remember that I edited the metadata but must have mixed up some of the dashes and spaces. Thanks for spotting that. Also thanks for the hints with the markdown markup. I will also change that in our JOSS draft! 

Closing the issue as it was fixed with https://github.com/cgre-aachen/gemgis_data/pull/2

**Comment by tarleb** [comment]
Happy to hear it worked :)

**Comment by AlexanderJuestel** [comment]
@tarleb, sorry to bother you again but somehow I managed to break the draft once again. I added an orcid number for the second author and changed some Information in the affiliations and now it is not compiling once again: 
https://github.com/cgre-aachen/gemgis_data/actions/runs/2467336434

I made sure that all spaces and dashes are correct as you mentioned that before.

The draft is stored here: 
https://github.com/cgre-aachen/gemgis_data/tree/main/jose

Thanks for your help!
